pages/index.js

```diff
@@ -1,22 +1,45 @@
 import Head from "next/head";
+import { useState } from "react";
 import styled from "styled-components";
+import TripCard from "../components/TripCard";
+import { dbTrips } from "../db";
 
 export default function Home() {
+  const [trips, setTrips] = useState(dbTrips);
   return (
     <div>
       <Head>
-        <title>My App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
```

Comment:
why did u remove those infos?

Reply:
I dont need page-specific meta tags. I'm gonna add them for the whole app in a later stage.